Logger/LogFactory.cs

```diff
@@ -1,10 +1,27 @@
-﻿namespace Logger;
+﻿using System;
+using System.Runtime.CompilerServices;
+
+namespace Logger;
 
 public class LogFactory
 {
-    public BaseLogger CreateLogger(string className)
+
+    private string? _pathName; 
+
+    public BaseLogger? CreateLogger(string className)
     {
+        if (_pathName == null)
+        {
+            return null;
+        }
 
-        return null;
+        return new FileLogger(_pathName) {
```

Comment:
Braces should be on a new line.
```suggestion
        return new FileLogger(_pathName) 
        {
```